dashboard/app/views/peer_reviews/show.html.haml

```diff
@@ -1,7 +1,7 @@
 - content_for(:head) do
   %script{src: minifiable_asset_path('js/plc.js')}
 .peer-review
-  - reviewed = !!@peer_review.status
+  - reviewed = @peer_review.escalated? ? @peer_review.reviewer_id? : @peer_review.status?
```

Comment:
I don't see where these methods come from - is there some rails magic happening here?

Reply:
🎩